react/AddonWrapper.js

```diff
@@ -0,0 +1,35 @@
+import React, { useContext } from 'react'
+import { ExtensionPoint, useRuntime } from 'vtex.render-runtime'
+import { ProductContext } from 'vtex.product-context'
+import { path } from 'ramda'
+import classNames from 'classnames'
+
+const AddonWrapper = () => {
+    const {
+        hints: { mobile },
+    } = useRuntime()
+    const valuesFromContext = useContext(ProductContext)
+    
+    const product= {
+        quantity: 1,
+        skuId: path(['selectedItem', 'itemId'], valuesFromContext),
+        productId: path(['product', 'productId'], valuesFromContext)
```

Comment:
Check the code of:
https://github.com/vtex-apps/product-summary/pull/164/files

You're creating a new `product` object in every render.